src/lib/plugin.js

```diff
@@ -130,7 +131,7 @@ class FiveBellsLedger extends EventEmitter2 {
 
     // Resolve ledger metadata
     const ledgerMetadata = yield this._fetchLedgerMetadata()
-    this.accountUriTemplate = ledgerMetadata.urls.account
+    this.urls = ledgerMetadata.urls
```

Comment:
You should sanitize this object, before making it part of the plugin's state. Something like this:

``` js
this.urls = {
  account_transfers: ledgerMetadata.account_transfers,
  //...
}

// Check required ledger endpoints are present
if (typeof this.urls.account_transfers !== 'string') {
  throw new Error('...')
}
```

It's better to fail when connecting if we can than at some random later time.